### DISCUSSION TITLE
Can I still use where when useColumns is false?

### DISCUSSION THREAD

**Discussion by ddxl123** [simolus3/drift#2050] [Q&A] [answered]
In `select( tableA )..innerJoin( tableB, useColumns: false)` , 
can `where( tableA.id.equals( aId ) )` or `where( tableB.id.equals( bId ) )` be used?

`useColumns` is false.

**Comment by simolus3** [comment, marked as answer]
I assume you meant `select(tableA).join([innerJoin( tableB, useColumns: false)])`, right?

But yes, you can still use columns in expressions including a `where` clause`. `includeColumns: false` only means that the column is not added to the `SELECT`, meaning that you can't read it from a result row.
